environments/dev/terraform/main.tf

```diff
@@ -36,7 +36,7 @@ module monster_infrastructure {
   }
 
   is_production = false
-  cluster_size  = 3
+  cluster_size  = 4
```

Comment:
"Bumps the # of nodes in the cluster to 4 since there won't be enough CPU to schedule both crons"
Just wondering how this is figured - do we know how much CPU each cron would require?

Reply:
@quazi-broad great question. Under the hood, the most CPU intensive step is transforming clinvar XML to json, here https://github.com/DataBiosphere/clinvar-ingest/blob/da1431fdb731b4a5af9c2fabff0ca43207422088/orchestration/templates/process-xml-release.yaml#L107-L108.

This requires "1500 cpu-m" units. When two instances of those are running, K8S reports that the pods are unschedulable due to limit CPU quota. This is a somewhat ham-fisted way of working around that.